packages/kbn-pm/dist/index.js

```diff
@@ -61,7 +61,7 @@ module.exports =
 /******/ 	__webpack_require__.p = "";
```

Comment:
__note:__ Bluebird added ~4000 lines to `dist/index.js` :)

Reply:
Shit happens. Hopefully we'll move to Node8 soon.